src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;

```

Comment:
Please, store a `Color` enum value instead of a string literal here.

Reply:
Also, make sure that you will process enum value instead of a string literal value across your solution.